tests/integration_tests/performance/test_block_performance.py

```diff
@@ -0,0 +1,541 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Performance benchmark for block device emulation."""
+import concurrent
+import json
+import logging
+import os
+from enum import Enum
+import shutil
+import pytest
+
+
+from conftest import _test_images_s3_bucket
+from framework.artifacts import ArtifactCollection, ArtifactSet
+from framework.builder import MicrovmBuilder
+from framework.matrix import TestContext, TestMatrix
+from framework.statistics import core, criteria
+from framework.utils import get_cpu_percent, CmdBuilder, eager_map, \
+    get_cpu_model_name
+import host_tools.drive as drive_tools
+import host_tools.network as net_tools  # pylint: disable=import-error
+import framework.statistics as st
+from integration_tests.performance.block_performance_test_config import \
```

Comment:
Tried it. Unfortunately, using `import *`, pylint will give us a warning. This could be avoided by adding this comment: `# pylint: disable=W0401`.

This is not enough, since pytlint also throws errors, stating that every indentifier in the imported file is undefined. We could mitigate by this adding another supression comment.

However, I wouldn't do it, since this would make pylint ignore any other undefined variable error (some that should actually be errors, e.g.: `print(sadasnjkldsfndsksncksd)`

Reply:
Just use `from integration_tests.performance.block_performance_test_config as test_config`.

Then you can use `test_config.FIO.something`.

Reply:
Also, as a rule of thumb, keep imported symbols to a minimum. If multiple symbols need to be imported from one file, use the import name as a prefix as stated above.

Otherwise we add too many symbols in the current module and their origins become hard to track.